### ISSUE TITLE
Footnotes are not highlighted by default in the editor

### ISSUE THREAD

**Issue by xibe** [seankwilliams/modern-footnotes#53]
Hi @seankwilliams , and thanks for this plugin which I'm happy to use for side-tracks and inline humor :)

With the latest release of WordPress (version 6.3), footnotes are not highlighted by default in the editor:

<img width="620" alt="image" src="https://github.com/seankwilliams/modern-footnotes/assets/2806515/e3f2c477-3195-4a53-b677-333599170bd6">

The author has to click the footnote in order to see how wide it is:

<img width="620" alt="image" src="https://github.com/seankwilliams/modern-footnotes/assets/2806515/f6ef01ab-b929-4495-a588-628b8c886660">

I don't know how hard or easy of a fix this could be, but it would be great to have that highlight back.

Thank you very much!

**Comment by seankwilliams** [comment]
Hi @xibe ,

Thanks for pointing this out! I'm trying to replicate, and I'm still seeing the highlighting in WordPress 6.3.1 in the post editor:

![image](https://github.com/seankwilliams/modern-footnotes/assets/151931/f0030c0f-cb3a-460d-8eae-3a05eacc1c3a)

Any ideas on how I could replicate? I see you have a different overall background color than I do in my editor, so I wonder if there's a different page or area I need to check.

**Comment by xibe** [comment]
Thank you for you quick reply, Sean!

I'm using 6.3.1 as well.

I'm using the Twenty Twenty theme, and the editor just applies its style -- I didn't change anything on that side.
I'm unsure whether this is tied to the theme, to Modern Footnotes or to WP itself, but what's certain is that the loss of non-select highlighted happened when I update to 6.3. 

I would need to install a test version of WP with various themes in order to check other contexts.

**Comment by xibe** [comment]
Here are my tests with various default themes, in WP trunk (6.4-alpha-56548):

2023:
<img width="1030" alt="image" src="https://github.com/seankwilliams/modern-footnotes/assets/2806515/627e7b80-8ad3-4d7a-acc9-ce6822fd7216">

2022:
<img width="1030" alt="image" src="https://github.com/seankwilliams/modern-footnotes/assets/2806515/624f17f3-75b7-4f1b-8983-65bf24d06de8">

2021:
<img width="1030" alt="image" src="https://github.com/seankwilliams/modern-footnotes/assets/2806515/b36eef29-cbd4-4635-bac0-501def56ea86">

2020:
<img width="1030" alt="image" src="https://github.com/seankwilliams/modern-footnotes/assets/2806515/05f5b363-e4de-4d34-8584-65759c1b6bf6">

2019:
<img width="1030" alt="image" src="https://github.com/seankwilliams/modern-footnotes/assets/2806515/e60f99ee-e96a-4df5-93f7-a7c9e903a4d4">

2017:
<img width="1030" alt="image" src="https://github.com/seankwilliams/modern-footnotes/assets/2806515/c072f4c0-781b-4f2e-b6c9-8c2e31eed306">

2016:
<img width="1030" alt="image" src="https://github.com/seankwilliams/modern-footnotes/assets/2806515/82966dd4-0f51-472a-a6aa-c71c7578577d">

and when selecting:
<img width="1030" alt="image" src="https://github.com/seankwilliams/modern-footnotes/assets/2806515/a65985ad-bb60-426b-ab87-34031f43f9a1">

In short, none of the latest default themes seem to highlight the footnotes by default anymore.

Next step would be to test with 6.2 :)

**Comment by seankwilliams** [comment]
Thanks @xibe ! I've been able to replicate this now. I'll work on a fix for this!

**Comment by xibe** [comment]
Fantastic, thanks a lot!

**Comment by xibe** [comment]
Hi Sean! Were you able to find a fix? Thank you very much!

**Comment by seankwilliams** [comment]
@xibe I've been able to replicate the issue, and fixing it is in my dev queue. I've had limited development bandwidth though, hence why it's taking a while. If you’re interested, I welcome any pull requests with a proposed fix. PRs are always welcome, and your contribution would be valuable.

**Comment by xibe** [comment]
Ah, I wish I could ([as I did in the past](https://core.trac.wordpress.org/ticket/13165)), but even my PHP5 chops are rusty at best :)

**Comment by seankwilliams** [comment]
All good! I'll get this fixed in one of the future updates.

**Comment by seankwilliams** [comment]
that is excellent news @xibe , thank you for updating me on that!
